tsdb/index/tsi1/series_file.go

```diff
@@ -257,17 +260,61 @@ func (f *SeriesFile) HasSeries(name []byte, tags models.Tags, buf []byte) bool {
 
 // SeriesCount returns the number of series.
 func (f *SeriesFile) SeriesCount() uint64 {
-	return uint64(f.hashMap.Len())
+	f.mu.RLock()
+	n := uint64(f.seriesMap.n + f.seriesMap.inmem.Len())
+	f.mu.RUnlock()
+	return n
 }
 
 // SeriesIterator returns an iterator over all the series.
 func (f *SeriesFile) SeriesIDIterator() SeriesIDIterator {
 	return &seriesFileIterator{
 		offset: 1,
-		data:   f.data[1:f.n],
+		data:   f.data[1:f.size],
 	}
 }
 
+func (f *SeriesFile) compactSeriesMap() error {
+	// TEMP: Compaction should occur in parallel.
```

Comment:
Can we change this to `TODO` so it's more searchable?

Reply:
Fixed.